### DISCUSSION TITLE
Http4s-Grpc Org Inclusion

### DISCUSSION THREAD

**Discussion by ChristopherDavenport** [http4s/http4s#7017] [General]
Http4s-Grpc is an Ember Only(currently, may work on other backends if we eventually get a different prior knowledge key for those backends) Grpc solution. That has a core module focusing on how to do grpc over http4s,  code-generation, a plugin to make everything simple for end users. 

As its very http4s related and is tied into the h2 capabilities of ember(currently), it seems like http4s may be a better communal home for the repository. 

https://github.com/davenverse/http4s-grpc

Thoughts?

**Comment by danicheg** [comment]
If it's `http4s-grpc` why it's not placed in the `http4s` organization? 😜 
\+ it'd be awesome to add a few examples, wdyt?

**Reply by ChristopherDavenport** [reply]
Working on examples. The codegen testing has a basic one, and I'm working on updating the grpc-playground.

**Reply by ChristopherDavenport** [reply]
https://github.com/ChristopherDavenport/grpc-playground - Here is is an updated example showing http4s-grpc.

**Comment by danicheg** [comment]
@armanbilge @ChristopherDavenport hey, let's proceed with this since we all agreed on the proposed transfer. I believe that some or both of you have the appropriate access to relocate the repository under the http4s organization https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-organization
Considering the precedent of the previous 'Great Schism,' it seems counterproductive to place this library under http4s/http4s. Surely, I can be wrong.

**Reply by armanbilge** [reply]
Done 🚀 https://github.com/http4s/http4s-grpc

@valencik is working on setting up a team (anyone who wants to join please let us know)

@danicheg would have bandwidth to switch the build to sbt-http4s-org? thank you 🙏

**Reply by valencik** [reply]
Looks like I have permissions to do this.
I agree it should be `http4s/http4s-grpc` (a new repo)

**Reply by hamnis** [reply]
I want to join the team.

**Comment by danicheg** [comment]
https://github.com/http4s/http4s-grpc
